landscaper/common/generate/openapi/generate.go

```diff
@@ -34,7 +34,8 @@ const filterPackagesDelimiter = ";"
 
 var (
 	// flags
-	outputDirectory, inputDirectory, rootDirectory, rootPackage, licensePath, verbosity, filterPackages string
+	outputDirectory, rootDirectory, licensePath, verbosity, filterPackages string
```

Comment:
`filterPackages` can be also a `StringSliceVar` similar to `inputDirectories` and `rootPackages`.